client/app/data/dashboard/bceid/my_operator.json

```diff
@@ -1,9 +1,13 @@
 [
   {
     "title": "My Operator",
+    "icon": "Inbox",
     "content": "View or update your operator information, which needs to be complete before applying for BORO ID's",
     "links": [
-      { "title": "My Operator ➤", "href": "/dashboard/select-operator" }
+      {
+        "href": "/dashboard/select-operator",
+        "notification": "industryOperator"
+      }
```

Comment:
Do you think the `notification` key is needed or is it safe to pass the key to the title?